Makefile

```diff
@@ -40,10 +40,18 @@ coverage:
 install:
 	go install ./cmd/rosa
 
+.PHONY: gci-install
+gci-install:
+	go install github.com/daixiang0/gci@v0.10.1
+
 .PHONY: fmt
-fmt:
+fmt: fmt-imports
 	gofmt -s -l -w cmd pkg
 
+.PHONY: fmt-imports
+fmt-imports: gci-install
+	find . -name '*.go' -not -path './vendor/*' | xargs gci write -s standard -s default -s "prefix(k8s)" -s "prefix(sigs.k8s)" -s "prefix(github.com)" -s "prefix(gitlab)" --custom-order --skip-generated
```

Comment:
@oriAdler My vote would be to add `-s "prefix(github.com/openshift/rosa)"` as the last `-s` arg to the `gci` call so that dependencies from with `rosa` are in their own section. I personally find it quite useful to be able to quickly identify the dependencies from within the rosa project that are being used.

That's just me. Let's see what other folk say and if I'm the only one that feels it's useful, we can ignore me :-)

Reply:
Sounds good to me

Reply:
Yeah I second that, if we can group the internal pkgs that would be awesome :+1:

Reply:
Added, thanks